hydrolib/core/io/mdu/models.py

```diff
@@ -177,11 +182,119 @@ class Lateral(INIBasedModel):
     _header: Literal["Lateral"] = "Lateral"
     id: str
     name: str = ""
+    locationType: Optional[str]
     nodeId: Optional[str]
     branchId: Optional[str]
     chainage: Optional[float]
+    numCoordinates: Optional[int]
+    xCoordinates: Optional[List[int]]
+    yCoordinates: Optional[List[int]]
```

Comment:
I believe that if there are one or multiple coordinates in the file, they still need to be split into a list, otherwise pydantic cannot make the conversion. You could take a look at the usages of the `get_split_string_on_delimiter_validator`.

Reply:
The validator is tested through for e.g. the test `test_split_string_strip_whitespace`.